packages/release-notes-generator/package.json

```diff
@@ -1,6 +1,16 @@
 {
 	"name": "@directus/release-notes-generator",
-	"private": true,
+	"version": "0.0.0",
```

Comment:
This is actually on purpose, the version is then set correctly by `changesets` on the first release. Otherwise the first released version would be, `0.0.2`. See also https://github.com/directus/directus/pull/18671.